charts/pega/README.md

```diff
@@ -356,15 +356,16 @@ ingress:
 
 You can optionally configure the resource allocation and limits for a tier using the following parameters. The default value is used if you do not specify an alternative value. See [Managing Kubernetes Resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) for more information about how Kubernetes manages resources.
 
-Parameter       | Description    | Default value
----             | ---       | ---
-`replicas`      | Specify the number of Pods to deploy in the tier. | `1`
-`cpuRequest`    | Initial CPU request for pods in the current tier.  | `2`
-`cpuLimit`      | CPU limit for pods in the current tier.  | `4`
-`memRequest`    | Initial memory request for pods in the current tier. | `6Gi`
-`memLimit`      | Memory limit for pods in the current tier. | `8Gi`
-`initialHeap`   | This specifies the initial heap size of the JVM.  | `4096m`
-`maxHeap`       | This specifies the maximum heap size of the JVM.  | `7168m`
+Parameter       | Description                                            | Default value
+---             | ---                                                    | ---
+`replicas`      | Specify the number of Pods to deploy in the tier.      | `1`
+`cpuRequest`    | Initial CPU request for pods in the current tier.      | `2`
+`cpuLimit`      | CPU limit for pods in the current tier.                | `4`
+`memRequest`    | Initial memory request for pods in the current tier.   | `10Gi`
+`memLimit`      | Memory limit for pods in the current tier.             | `10Gi`
+`initialHeap`   | This specifies the initial heap size of the JVM.       | `4096m`
```

Comment:
make these consistent with line 361>>
Specify the initial heap size of the JVM.       | `4096m`
Specify the maximum heap size of the JVM.       | `8192m`
Specify the maximum metaspace size of the JVM.

Reply:
Should be good now